src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/WorkflowComponent.scala

```diff
@@ -232,6 +232,17 @@ trait WorkflowComponent {
       UpdateWorkflowStatusRawSql.actionForCurrentStatusAndSubmission(submissionId, currentStatus, newStatus)
     }
 
+    def findActiveWorkflowsWithoutExternalIds(workspaceContext: SlickWorkspaceContext): ReadAction[Seq[WorkflowRecord]] = {
+      findWorkflowsByWorkspace(workspaceContext).filter(!_.externalId.isDefined).result map {workflowRecords =>
+        workflowRecords.filter(wf => !WorkflowStatuses.withName(wf.status).isDone)
```

Comment:
this (and line 243) are filtering in memory but probably should filter in the database